### PR TITLE
Hide latest notification on clickthrough

### PR DESCRIPTION
**Pull request by phildarnowsky**
Card: https://app.asana.com/0/1148853526253426/1199581932380738

### CODE REVIEW

File: assets/tests/components/notifications.test.tsx [hunk 1]
```diff
@@ -257,12 +252,29 @@ describe("NotificationCard", () => {
...
+        currentTime={currentTime}
+        openVPPForCurrentVehicle={openVPPForCurrentVehicle}
+      />
+    )
+    expect(openVPPForCurrentVehicle).not.toHaveBeenCalled()
+    expect(dispatch).not.toHaveBeenCalled()
+    wrapper.find(".m-notifications__card-info").simulate("click")
+    expect(openVPPForCurrentVehicle).toHaveBeenCalled()
+    expect(dispatch).toHaveBeenCalledWith({ type: "HIDE_LATEST_NOTIFICATION" })
+    /*    mockUseReducerOnce(notificationsReducer, {
```
Comment (lemald) [inline assets/tests/components/notifications.test.tsx:277]: I think you meant to delete (rather than comment) this code?

Comment (phildarnowsky) [reply assets/tests/components/notifications.test.tsx:277]: I sure did.

**Comment by phildarnowsky** [comment]
@lemald ready for another look

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**